tests/tron/test_tron_llama1b_train.py

```diff
@@ -118,6 +118,7 @@ def test_mock_training_checkpoint(self, tmp_path):
                 logger_config=LoggerConfig(
                     log_interval=1,
                     tensorboard_dir=tensorboard_dir,
+                    logging_level=20,
```

Comment:
change the default here instead:

https://github.com/NVIDIA/NeMo/blob/b838fa061c8872626b313aef06b3cd4c6a2821ef/nemo/tron/config.py#L331-L332

it shouldn't be `Optional`. we can default it to `logging.INFO`

otherwise we get the failure here about an invalid log level: https://github.com/NVIDIA/NeMo/blob/b838fa061c8872626b313aef06b3cd4c6a2821ef/nemo/tron/utils/log_utils.py#L75